config.md

```diff
@@ -27,13 +27,15 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
 * **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+* **`autoremove`** (bool, optional) If true then the container will be automatically deleted once the PID namespace is deleted. This removes the need for an explicit call to the `delete` operation. Defaults to false.
```

Comment:
On Wed, May 11, 2016 at 10:24:13PM -0700, Qiang Huang wrote:
> > once the PID namespace is deleted
> 
> Isn't this too technical detailed? What if a container doesn't have
> it's own PID namespace?  Is we call it `processes of the container
> are exited` sufficient enough?

I think we may be dropping ‘autoremove’ after today's meeting [1].
But if we keep it, I like your wording better than the current
PID-namespace reference.  I'd take it one step further and use a
singular “container process” though, since it's easy to wait on a
process that you clone(2) into existence, but hard to defining
“container processes” plural is complicated [2].

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2016/opencontainers.2016-05-11-17.03.log.html#l-38
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/n7vxlhn6Q7g
     Subject: Definining "container" and "container processes" on Linux
     Date: Thu, 8 Oct 2015 09:47:08 -0700
     Message-ID: <20151008164708.GM28418@odin.tremily.us>